src/Aspire.Hosting.Kafka/KafkaServerResource.cs

```diff
@@ -12,14 +12,21 @@ namespace Aspire.Hosting;
 public class KafkaServerResource(string name) : ContainerResource(name), IResourceWithConnectionString, IResourceWithEnvironment
 {
     internal const string PrimaryEndpointName = "tcp";
+    internal const string InternalEndpointName = "internal";
```

Comment:
Add a comment explaining what each of these endpoints represent.

Reply:
done. Extended public xml doc too.